### ISSUE TITLE
Fix Codacy errors in 0.1.0 branch

### ISSUE THREAD

**Issue by chupman** [JanusGraph/janusgraph-python#22]
Once the initial PR is merged create a follow-up PR that resolves the issue listed by codacy.

**Comment by FlorianHockmann** [comment]
Closing as it's no longer valid now that we have merged a first version of the library that is not based on these initial PRs.
